src/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerFactoryExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging.EventSourceLogger;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <summary>
+    /// Extension methods for the <see cref="ILoggerFactory"/> class.
+    /// </summary>
+    public static class EventSourceLoggerFactoryExtensions
+    {
+        /// <summary>
+        /// Adds an event logger that is enabled for <see cref="LogLevel"/>.Information or higher.
+        /// </summary>
+        /// <param name="factory">The extension method argument.</param>
+        public static ILoggerFactory AddEventSourceLogger(this ILoggerFactory factory)
```

Comment:
public static ILoggerFactory AddEventSourceLogger(this ILoggerFactory factory**, string spec = null**)
How about we pass spec as a parameter here?